joule-labs/joule-extension
# Tx testnet info

Closes #201 

### Added handling for Blockstream Bitcoin Testnet Explorer

Check for ```testnet === true``` in the node.nodeInfo. Added testnet URL to on-chain transaction info as well as funding tx id for channel info. 

### Steps to Test

1. Start up LND on testnet
2. Open a channel and verify clicking on channel info/funding tx id link opens on Blockstream Explorer
3. Click on transactions and verify both block height and tx hash links open on Blockstream Explorer
